lib/db/schema/patch-026-027.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS securityEvents (
+  uid BINARY(16) NOT NULL,
+  name VARCHAR(255) NOT NULL,
+  ipAddr VARBINARY(16) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL
+) ENGINE=InnoDB;
```

Comment:
What should be the primary key on this table, and does it need any secondary indexes?  My suggestion would be a composite primary key of `(uid, createdAt)` so that you can easily fetch the events for a particular user, and easily page through them in timestamp order.